.github/files/build-reminder-comment/check-test-reminder-comment.js

```diff
@@ -1,56 +1,37 @@
-const deFusionedFiles = require( './de-fusioned-files.js' );
 const getCheckComments = require( './get-check-comments.js' );
 
 /* global GitHub, Core */
 
 /**
- * Does the PR touches any files that have been De-Fusioned?
+ * Does the PR touches anything that needs testing on WordPress.com.
+ *
+ * Currently we look whether process.env.CHANGED contains `plugins/jetpack`,
+ * meaning that Jetpack is being built.
  *
  * @param {GitHub} github  - Initialized Octokit REST client.
  * @param {string} owner   - Repository owner.
  * @param {string} repo    - Repository name.
  * @param {string} number  - PR number.
  * @param {Core}   core    - A reference to the @actions/core package
- * @returns {Promise<boolean>} Promise resolving to a boolean if the PR touches Fusioned files.
+ * @returns {Promise<boolean>} Promise resolving to a boolean if the PR touches something needing testing.
```

Comment:
```suggestion
 * @returns {Promise<boolean>} Promise resolving to a boolean if the PR touches something that needs testing.
```
Does this read clearer? (same for below).

Reply:
Seems the same to me. :shrug: